op-service/txmgr/metrics/tx_metrics.go

```diff
@@ -0,0 +1,32 @@
+package metrics
+
+import (
+	"github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type TxMetricer interface {
+	RecordL1GasFee(receipt *types.Receipt)
+}
+
+type TxMetrics struct {
+	TxL1GasFee prometheus.Gauge
+}
+
+var _ TxMetricer = (*TxMetrics)(nil)
+
+func MakeTxMetrics(ns string, factory metrics.Factory) TxMetrics {
+	return TxMetrics{
+		TxL1GasFee: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: ns,
+			Name:      "tx_l1gas_fee",
```

Comment:
```suggestion
			Name:      "tx_fee",
```

Reply:
@trianglesphere so with metrics we are specifically emitting L1 fee and that's why I mentioned l1 in the name to be more clear. Could do maybe `tx_l1_fee` ?